lib/ws-channel.js

```diff
@@ -425,11 +487,11 @@ WebsocketChannel.create = function(onRequest) {
 };
 
 
-// Use to connect to a serveer, token/uri must be known from the server
+// Use to connect to a server, clientToken/uri must be known from the server
 // (communicated out of band).
 //
 // onRequest, receive requests/notifications from server
 // uri, ws://[token@]host:port/path
-WebsocketChannel.connect = function(onRequest, uri) {
-  return new WebsocketChannel(onRequest).connect(uri);
+WebsocketChannel.connect = function(onRequest, uri, options) {
```

Comment:
I don't think we need options, the URI will contain the clientToken, nothing else is needed to connect.